README.md

```diff
@@ -1,11 +1,31 @@
-# How The Web Works
+# How the Web Works
 
 ## Learning Goals
 
-- Define a client and server
-- Define a static site vs. a dynamic site
-- Explain what an HTTP request is
-- Explain the nature of request and response
+- Describe the components of a web application framework.
+- Manipulate and test the structure of a request object.
+
+***
+
+## Key Vocab
+
+- **Web Framework**: software that is designed to support the development of
+  web applications. Web frameworks provide built-in tools for generating web
+  servers, turning Python objects into HTML, and more.
+- **Extension**: a package or module that adds functionality to a Flask
```

Comment:
Do we mean to have the word "Flask" in here?

Reply:
oops, thanks!

Reply:
Relatedly, do you mean to have Python in the line above?

Reply:
thanks, replaced with java